src/python/ovms_py_tensor.cpp

```diff
@@ -26,6 +26,13 @@
 namespace py = pybind11;
 using namespace ovms;
 
+OvmsPyTensor::OvmsPyTensor(void* dataToCopy, const std::string& name, const std::vector<py::ssize_t>& shape, const std::string& datatype, py::ssize_t size) :
+    OvmsPyTensor(name, nullptr, shape, datatype, size) {
+    allocatedPtr = std::make_unique<char[]>(size);
+    ptr = this->allocatedPtr.get();
+    memcpy(ptr, dataToCopy, size);
```

Comment:
It feels more natural to me if order of that operations is switched - first copy then assignment. Can memcpy fail somehow?